_data/versions.yml

```diff
@@ -1,3 +1,3 @@
-java_sdk: 0.12.43
+java_sdk: 0.12.47
```

Comment:
It's used to show the correct SDK number in the documentation, and to link to that version's javadocs. Every version's javadocs are posted to S3, so it's okay if this lags behind the most recent version... but as features get introduced, we want to point the documentation toward newer versions of the SDK. So periodically I bump this up.